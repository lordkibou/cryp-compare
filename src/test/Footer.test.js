import renderer from 'react-test-renderer';
import Footer from '../scenes/footer'
import { BrowserRouter } from 'react-router-dom';
//If the change is expected you can invoke Jest with jest -u to overwrite the existing snapshot.

it('Testing Footer Scene', () => {
  const component = renderer.create(
    <BrowserRouter>
        <Footer/>
    </BrowserRouter>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});