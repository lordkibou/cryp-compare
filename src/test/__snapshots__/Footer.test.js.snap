// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Footer Scene 1`] = `
<footer
  className="footer undefined"
>
  <div>
    © 2023 Cryp Compare. All Rights Reserved.
  </div>
  <div
    className="footer-content"
  >
    <a
      className="link-text"
      href="https://github.com/lordkibou"
      rel="noopener noreferrer"
      target="_blank"
    >
      Made with 
      <span
        aria-label="heart"
        role="img"
      >
        ❤
      </span>
       by Juan Diaz
    </a>
  </div>
</footer>
`;
